web-common/src/features/dashboards/state-managers/state-managers.ts

```diff
@@ -124,6 +130,10 @@ export function createStateManagers({
     contextColWidthDefaults,
   );
 
+  // TODO: once we move everything from dashboard-stores to here, we can get rid of the global
+  initPersistentDashboardStore(metricsViewName + (extraKey ?? ""));
```

Comment:
I'd expect this to be org-project-dashboard, not dashboard-org-project, to match our page route hierarchy

Reply:
Not sure i understand. Is this comment in the wrong place?